epochalyst/pipeline/model/training/torch_trainer.py

```diff
@@ -825,6 +829,31 @@ def get_model_checkpoint_path(self, epoch: int) -> Path:
         """
         return Path(f"{self.trained_models_directory}/{self.get_hash()}_checkpoint_{epoch}.pt")
 
+    def _add_logging_prefix_postfix(self, message: dict[str, Any]) -> dict[str, Any]:
+        """Add logging prefix and postfix to the message."""
+        new_message = {}
+
+        # for normal messages, add prefix and postfix to every key
+        if message.get("type") != "wandb_plot":
+            for key, value in message.items():
+                new_message[f"{self.logging_prefix}{key}{self.logging_postfix}"] = value
+
+        # for wandb plots, add prefix and postfix to title only
+        else:
+            new_message = copy.deepcopy(message)
+            new_message["data"]["title"] = f"{self.logging_prefix}{new_message['data']['title']}{self.logging_postfix}"
```

Comment:
You probably want to swap the if condition
```suggestion
        if message.get("type") == "wandb_plot":
            new_message = copy.deepcopy(message)
            new_message["data"]["title"] = f"{self.logging_prefix}{new_message['data']['title']}{self.logging_postfix}"
     
        else:
            for key, value in message.items():
                new_message[f"{self.logging_prefix}{key}{self.logging_postfix}"] = value
```